src/SprykerEco/Zed/Computop/Business/Payment/Handler/Saver/Init/PaydirektResponseSaver.php

```diff
@@ -19,15 +19,15 @@ class PaydirektResponseSaver extends AbstractResponseSaver
      */
     public function save(QuoteTransfer $quoteTransfer): QuoteTransfer
     {
-        $responseTransfer = $quoteTransfer->getPayment()->getComputopPaydirekt()->getPaydirektInitResponse();
-        $this->setPaymentEntity($responseTransfer->getHeader()->getTransId());
-        if ($responseTransfer->getHeader()->getIsSuccess()) {
+        $computopPaydirektInitResponseTransfer = $quoteTransfer->getPaymentOrFail()->getComputopPaydirektOrFail()->getPaydirektInitResponseOrFail();
```

Comment:
required transfer properties should be mentioned at API specs

Reply:
Fixed